views/group/index.pug

```diff
@@ -43,7 +43,7 @@ block content
               //- Heroicon name: external-link
               svg(xmlns='http://www.w3.org/2000/svg' alt="Külső link" aria-label="Külső link" fill='none' viewbox='0 0 24 24' stroke='currentColor' class='w-6 h-6')
                 path(stroke-linecap='round' stroke-linejoin='round' stroke-width='2' d='M10 6H6a2 2 0 00-2 2v10a2 2 0 002 2h10a2 2 0 002-2v-4M14 4h6m0 0v6m0-6L10 14')
-              a(href=group.link class='text-lg sm:text-xl')= group.link
+              a(href=group.link class='text-lg sm:text-xl' target='_blank' rel='noopener')= group.link
```

Comment:
Maybe we could put a `noreferrer` to these external links too. [(source of tip)](https://clever-solution.com/everything-you-need-to-know-about-rel-noopener-noreferrer-tags-purpose-benefits-and-seo-impact/)

Reply:
We should have `noopener noreferrer` added to all external links.